File: Python/백준/Silver/7568. 덩치/덩치.py
a = int(input())
aa = []
bb = []
g = 0
for i in range(a):
    b,c = map(int,input().split())
    aa.append([b,c])
for i in range(a):
    s = 1
    for j in range(a):
        if aa[i][0]<aa[j][0] and aa[i][1]<aa[j][1]:
            s+=1
    bb.append(s)
print(*bb)

File: C#/백준/Silver/7568. 덩치/덩치.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int[,] aa = new int[a,2];
            for (int i=0;i<a;i++){
                int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                aa[i,0] = bb[0];
                aa[i,1] = bb[1];
            }
            for (int i=0;i<a;i++){
                int s = 1;
                for  (int j=0;j<a;j++){
                    if(aa[i,0]<aa[j,0] && aa[i,1]<aa[j,1]){
                        s++;
                    }
                }
                Console.Write($"{s} ");
            }
        }
    }
}